icons/valorant.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Valorant</title><path d="M23.792 2.1524a.25199.25199 0 0 0-.098.083c-3.3839 4.2298-6.7687 8.4597-10.15 12.689-.107.094-.025.28899.119.26599 2.4389.003 4.8768 0 7.3157.001a.65997.65997 0 0 0 .55198-.25c.77397-.96695 1.5499-1.9339 2.3239-2.9029a.71997.71997 0 0 0 .14399-.49097c-.002-3.0769.001-6.1528-.003-9.2286.016-.11-.1-.207-.20399-.168zm-23.715.014c-.077.038-.074.13199-.076.20499.002 3.0739.001 6.1498.001 9.2246a.67897.67897 0 0 0 .158.46298l7.6397 9.5496c.12.15199.30798.24999.50497.24699 2.4549 0 4.9098.003 7.3657 0 .142.021.222-.17399.116-.26499-5.1258-6.4147-10.26-12.825-15.387-19.24-.08-.094-.17399-.27199-.32199-.18399z"/></svg>
```

Comment:
It's sufficient to use the default precision of 3 in SVGOMG (see [contribution guidelines](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#3-optimize-the-icon) because there's almost no loss in quality.

```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Valorant</title><path d="M23.792 2.152a.252.252 0 0 0-.098.083c-3.384 4.23-6.769 8.46-10.15 12.69-.107.093-.025.288.119.265 2.439.003 4.877 0 7.316.001a.66.66 0 0 0 .552-.25c.774-.967 1.55-1.934 2.324-2.903a.72.72 0 0 0 .144-.49c-.002-3.077 0-6.153-.003-9.23.016-.11-.1-.206-.204-.167zM.077 2.166c-.077.038-.074.132-.076.205.002 3.074.001 6.15.001 9.225a.679.679 0 0 0 .158.463l7.64 9.55c.12.152.308.25.505.247 2.455 0 4.91.003 7.365 0 .142.02.222-.174.116-.265C10.661 15.176 5.526 8.766.4 2.35c-.08-.094-.174-.272-.322-.184z"/></svg>
```